.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,24 @@
-### What change does this PR introduce?
+### Summary
 
-<!-- Explain here the changes your PR introduces and text to help us understand the context of this change. -->
+<!-- Provide a concise summary "Why are the changes needed"? 
+Include any relevant links, such as Linear tickets, Slack discussions, 
+or design documents. -->
 
-### Why was this change needed?
+### Changes Made
 
-<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing, Example: Closes #31 -->
+<!-- Provide details on the approach taken to address the problem 
+and any notable implementation details. -->
 
-### Other information (Screenshots)
+### Screenshots
+<!-- If the changes are visual, include screenshots or GIFs. -->
```

Comment:
```suggestion
<!-- If the changes are visual, include screenshots or screencasts. -->
```

Reply:
I would say that for the OS community it should be required to attach the screenshots. We've been so many times in the past when someone created the PR and said that the visual issue was fixed, but then after checking the PR locally, it wasn't.

Reply:
But some PRs don't have any visual - like BE work. Thats why its phrased as optional. 
I suggest that when there are visual changes, but no screenshot is included, then we ask to please add screenshots before we can start reviewing their PR.
Also, now that there is a dedicated section for that, they will be more likely to include them IMO